app/templates/ui/app/search/search-results.html

```diff
@@ -1,6 +1,6 @@
 <div ng-repeat="result in results">
   <h4>
-    <a ui-sref="root.view({uri: result.uri})">{{ result.uri }}</a>
+    <a ui-sref="root.view({uri: result.uri})">{{ result.title || result.uri }}</a>
```

Comment:
I've previously used transforms that pass through most (or all) of the
search results and add in a title that is appropriate for the dataset.  I
could write that up in more detail, or we could add a transform to this
application.  Opinions?

On Tue, Sep 1, 2015 at 1:55 PM, Patrick McElwee <notifications@github.com>
wrote:

> In app/templates/ui/app/search/search-results.html
> <https://github.com/marklogic/slush-marklogic-node/pull/130#discussion_r38463651>
> :
>
> > @@ -1,6 +1,6 @@
> >  <div ng-repeat="result in results">
> >    <h4>
> > -    <a ui-sref="root.view({uri: result.uri})">{{ result.uri }}</a>
> > +    <a ui-sref="root.view({uri: result.uri})">{{ result.title || result.uri }}</a>
>
> I vote you show me ... erm ... them how to supply a title (maybe just a
> comment here, or maybe a recipe in the docs?)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/marklogic/slush-marklogic-node/pull/130/files#r38463651>
> .
>